packages/runtime/src/utils/load.ts

```diff
@@ -1,135 +1,67 @@
-import {
-  composeKeyWithSeparator,
-  loadScript,
-  loadScriptNode,
-  CreateScriptHookReturn,
-} from '@module-federation/sdk';
-import { assert } from '../utils/logger';
-import { getRemoteEntryExports, globalLoading } from '../global';
-import { Remote, RemoteEntryExports, RemoteInfo } from '../type';
+import { composeKeyWithSeparator, isBrowserEnv } from '@module-federation/sdk';
 import { DEFAULT_REMOTE_TYPE, DEFAULT_SCOPE } from '../constant';
+import { globalLoading } from '../global';
+import { Remote, RemoteEntryExports, RemoteInfo } from '../type';
+import { FederationHost } from '../core';
+import { loadEntryNode } from '../plugins/node/loadEntry';
```

Comment:
Because i see you no apply the `domPlugin` , so the utility function can move here

Reply:
should we remove `domPlugin` (and `nodePlugin`) altogether and just leave the hook there so users can override any way they want? I can move all of the stuff back into `packages/runtime/src/utils/load.ts` then.

Reply:
moved all of the utils back to `load.ts` and removed the now unnecessary plugins